libretroshare/src/serialiser/rsserial.cc

```diff
@@ -64,7 +64,7 @@ RsItem::RsItem(uint32_t t)
 class Counter
 {
 	public: 
-		Counter(int i): _i(i) {}
+		static Counter(int i): _i(i) {}
```

Comment:
static? It should be explicit. I'm curious whether that actually compiles, and if so, what it means!